### PR TITLE
Servicebus topic TF

### PR DESCRIPTION
**Pull request by vijayrajagopalan-hmcts**
### JIRA link (if applicable) ###
Upgrade module to use Terraform resource

### Change description ###
Upgrade module to use Terraform resource

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Contribution guidelines
```
Comment (timja) [inline .github/CONTRIBUTING.md:1]: all the same comments from https://github.com/hmcts/terraform-module-servicebus-namespace/pull/17 apply throughout this PR

**Review by adusumillipraveen — COMMENTED** [review]
Being addressed in Jenkins library to auto import

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (vijayrajagopalan-hmcts) [reply .github/CONTRIBUTING.md]: Deleted unwanted files

File: README.md [hunk 2]
```diff
@@ -1,3 +1,39 @@
...
+
+[Azure Service Bus Topic](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-queues-topics-subscriptions) <br />
+
+## Usage
+
+The following example shows how to use the module to create an Azure Service Bus topic. 
+
+```terraform
+module "servicebus-topic" {
+  source                = "git@github.com:hmcts/terraform-module-servicebus-topic?ref=servicebus_topic_tf"
```
[the remark this answers is not in the document]
Comment (vijayrajagopalan-hmcts) [reply README.md:14]: @timja if we immediately merge it with master, the existing service bus resources will get affected. Suggest the changes are merged with master after the import is executed. I have built a method in Jenkins Library, which imports template deployment resources into TF native resources. PR is hmcts/cnp-jenkins-library#721.

Praveen is reviewing this PR.

**Review by timja — APPROVED** [review]

**Review by adusumillipraveen — APPROVED** [review]
Please merge after Jenkins PR

**Comment by vijayrajagopalan-hmcts** [comment]
This PR will be merged after the issues in AAT are resolved to ensure the import and build executes successfully.

@timja @adusumillipraveen

**Pull request merged**